src/release/main.d

```diff
@@ -335,6 +332,28 @@ ActionList prepareMinorRelease ( ref HTTPConnection con, ref Repository repo,
 
 /*******************************************************************************
 
+    Prepares the actions to clar the release notes
+
+    Returns:
+        actions required to clear the release notes
+
+*******************************************************************************/
+
+ActionList clearReleaseNotes ( )
```

Comment:
Yeah, that is what we have been doing so far (though in wording of "base on vX.Y.Z release of previous major")